src/ocamlorg_package/lib/ocamlorg_package.ml

```diff
@@ -303,8 +303,11 @@ let documentation_page ~kind t path =
 
 let readme_file ~kind t =
   let open Lwt.Syntax in
-  let+ doc = documentation_page ~kind t "README.md.html" in
-  match doc with None -> None | Some { content; _ } -> Some content
+  let+ doc_md = documentation_page ~kind t "README.md.html"
+  and+ doc_org = documentation_page ~kind t "README.org.html" in
```

Comment:
Is it possible to know what `README` file to look for without making two requests?